tests/testthat/_snaps/as_tibble.md

```diff
@@ -0,0 +1,162 @@
+# as_tibble works with standard use
+
+    Code
+      as.data.frame(res)
+    Output
+             **Characteristic**       **N = 200**
+      1  Chemotherapy Treatment              <NA>
+      2                  Drug A          98 (49%)
+      3                  Drug B         102 (51%)
+      4                     Age       47 (38, 57)
+      5                 Unknown                11
+      6    Marker Level (ng/mL) 0.64 (0.22, 1.41)
+      7                 Unknown                10
+      8                 T Stage              <NA>
+      9                      T1          53 (27%)
+      10                     T2          54 (27%)
+      11                     T3          43 (22%)
+      12                     T4          50 (25%)
+      13                  Grade              <NA>
+      14                      I          68 (34%)
+      15                     II          68 (34%)
+      16                    III          64 (32%)
+      17         Tumor Response          61 (32%)
+      18                Unknown                 7
+      19           Patient Died         112 (56%)
+      20 Months to Death/Censor 22.4 (15.9, 24.0)
+
+---
+
+    Code
+      as.data.frame(res)
+    Output
+        **Characteristic** **OR** **95% CI** **p-value**
+      1                Age   1.02 1.00, 1.04        0.10
+      2              Grade   <NA>       <NA>        <NA>
+      3                  I   <NA>       <NA>        <NA>
+      4                 II   0.85 0.39, 1.85         0.7
+      5                III   1.01 0.47, 2.16        >0.9
+
+---
+
+    Code
+      as.data.frame(res)
+    Output
+        **Characteristic** **N** **OR** **95% CI** **p-value**
+      1                Age   183   1.02 1.00, 1.04        0.10
+      2              Grade   193   <NA>       <NA>        <NA>
+      3                  I  <NA>   <NA>       <NA>        <NA>
+      4                 II  <NA>   0.95 0.45, 2.00         0.9
+      5                III  <NA>   1.10 0.52, 2.29         0.8
+
+# as_tibble works with bold/italics
```

Comment:
Same comment as above. I think we can do the same test but with a smaller table.